libs/parallelism/algorithms/include_compatibility/hpx/parallel/traits/projected.hpp

```diff
@@ -10,12 +10,16 @@
 #include <hpx/algorithms/config/defines.hpp>
 #include <hpx/algorithm.hpp>
 
+// different versions of clang-format produce different results
```

Comment:
Isn't this why we specify which version we use in CI?

Reply:
Yes, but it's difficult to always stay in sync with the version we use in CI.